aptos-move/framework/aptos-framework/sources/state_storage.move

```diff
@@ -23,54 +21,40 @@ module aptos_framework::state_storage {
         usage: Usage,
     }
 
-    /// This updates at reconfig and guarantees not to change elsewhere, safe
-    /// for gas calculation.
-    ///
-    /// Specifically, it copies the usage at the begining of the concluding
-    /// epoch for gas calculation of the entire next epoch. -- The data is one
-    /// epoch older than ideal, but the Vm doesn't need to worry about reloading
-    /// gas parameters after the first txn of an epoch.
-    struct GasParameter has key, store {
```

Comment:
You can't remove this without reset of the chain.

Reply:
That's something we'll discuss with @davidiw ... Yeah, I guess I have to keep and put it to the end of the file for now tho.